tests/test_settings_api.py

```diff
@@ -139,3 +140,71 @@ def test_api_upgrade(new_solver_session, capsys):
     case_path = download_file("Static_Mixer_main.cas.h5", "pyfluent/static_mixer")
     solver.tui.file.read_case(case_path)
     "<solver_session>.file.read_case" in capsys.readouterr().out
+
+
+@pytest.mark.skip(reason="Skipping till docker image is updated")
```

Comment:
As there is a significant change in flobject.py, I'll merge bith Fluent and PyFluent PR at the same time. I'll enable the new test after Fluent docker image is updated.

Reply:
I a;so need to push a dev version as the new PyConsole implmentation will not work with newer Fluent + older PyFluent.